iothub/service/src/Registry/Models/ExportImportDevice.cs

```diff
@@ -168,7 +168,7 @@ public ExportImportDevice(Device device, ImportMode importmode)
         /// For more information, see
         /// <see href="https://docs.microsoft.com/azure/iot-edge/iot-edge-as-gateway?view=iotedge-2020-11#parent-and-child-relationships"/>.
         /// </remarks>
-        [JsonPropertyName("deviceScope", NullValueHandling = NullValueHandling.Include)]
```

Comment:
Does System.Text.Json support NullValueHandling.Ignore? That seems like a case that we might want to use in some cases.

Reply:
No, but you can get it by using `JsonSerializerOptions`, however, I think we probably shouldn't use it as it is how a user can explicitly unset a value.